ridewave/app/api/tenants/[id]/subscription/route.ts

```
@@ -0,0 +1,304 @@
+            { status: 400 }
+          );
+        }
+
+        // Create Stripe checkout session
+        const checkoutUrl = await SubscriptionService.createCheckoutSession(
+          tenantId,
+          validatedData.tier,
+          validatedData.billingCycle,
+          validatedData.successUrl || `${request.nextUrl.origin}/dashboard/billing?success=true`,
+          validatedData.cancelUrl || `${request.nextUrl.origin}/dashboard/billing?canceled=true`
+        );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)